feedstock-patches/pyarrow/s390x-fix1.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/recipe/meta.yaml b/recipe/meta.yaml
```

Comment:
I don't think this file is needed.

Reply:
yeah. I added these changes to the other PR on top of my earlier PR. Will remove this file.